table.cpp

```diff
@@ -109,6 +102,9 @@ TableCell::TableCell(float d, const std::string& c, int precision, const std::st
   } else if (event == "Extension"){
     extension = true;
     inquiry = bad_status = override = version_conflict = cancelled = false;
+    if (reason != ""){
+    hoverText = "class=\"hoverable-cell\" data-hover-text=\"" + userName + " received a " + std::to_string(daysExtended) + " day extension due to " + reason + " on " + gID + "\" ";
```

Comment:
this line should be indented

Reply:
Fixed in [a7eefe9](https://github.com/Submitty/RainbowGrades/pull/77/commits/a7eefe9949131c95c69593444fb463806ced3a3f)